ui/components/component-library/avatar-network/avatar-network.js

```diff
@@ -0,0 +1,95 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { BaseAvatar } from '../base-avatar';
+
+import {
+  COLORS,
+  SIZES,
+  DISPLAY,
+  ALIGN_ITEMS,
+  JUSTIFY_CONTENT,
+} from '../../../helpers/constants/design-system';
+
+export const AvatarNetwork = ({
+  size = SIZES.MD,
+  backgroundColor = COLORS.BACKGROUND_ALTERNATIVE,
+  borderColor = COLORS.BORDER_DEFAULT,
+  networkName,
+  networkImageUrl,
+  showHalo,
+  ...props
+}) => {
+  const [showFallback, setShowFallback] = useState(false);
+  useEffect(() => {
+    setShowFallback(!networkImageUrl);
+  }, [networkImageUrl]);
+
+  const baseStyle = showHalo ? { position: `relative` } : {};
+  const sizePercentage = showHalo ? '62.5%' : '100%';
+
+  const style = size
+    ? {
+        height: sizePercentage,
+        width: sizePercentage,
+        borderRadius: '50%',
+        zIndex: '2',
+      }
+    : {};
+
+  const handleOnError = () => {
+    setShowFallback(true);
+  };
+
+  return (
+    <BaseAvatar
+      size={size}
+      style={baseStyle}
+      display={DISPLAY.FLEX}
+      alignItems={ALIGN_ITEMS.CENTER}
+      justifyContent={JUSTIFY_CONTENT.CENTER}
+      data-test-id="avatar-network"
+      {...{ backgroundColor, borderColor, ...props }}
+    >
+      {showFallback ? (
+        networkName?.[0].toUpperCase() ?? '?'
```

Comment:
Imo both are fine, if I replace it with ```networkName?.[0]?.toUpperCase()```  it will handle the edge case if blank string is passed. I am avoiding using css since I have to replace it with the Typography component very soon. Lmk if using it with css still seems better @garrettbear ?